### ISSUE TITLE
User cannot identify broken template from the log

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#2038]
__Describe the Bug__

Error message in the log does not contain any information which would allow to identify the broken template.

<details><summary>log</summary>

```
[/Users/maciej/Desktop/diagram_1.bpmn] invalid property.binding type <undefined>; must be any of { property, camunda:property, camunda:inputParameter, camunda:outputParameter, camunda:in, camunda:out, camunda:in:businessKey, camunda:executionListener, camunda:field }
    at Validator._logError (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:239:13)
    at Validator._validateProperty (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:197:18)
    at Array.filter (<anonymous>)
    at Validator._validateProperties (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:161:38)
    at Validator._validateTemplate (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:114:17)
    at Validator.add (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:54:20)
    at Array.forEach (<anonymous>)
    at Validator.addAll (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:39:17)
    at ElementTemplatesLoader.setTemplates (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/ElementTemplatesLoader.js:59:35)
    at BpmnEditor.loadTemplates (webpack-internal:///./src/app/tabs/bpmn/BpmnEditor.js:417:21) [ error ]
This error may be the result of a plug-in compatibility issue. [ info ]
Disable plug-ins (restarts the app) [ info ]
[/Users/maciej/Desktop/diagram_1.bpmn] invalid property.binding type <undefined>; must be any of { property, camunda:property, camunda:inputParameter, camunda:outputParameter, camunda:in, camunda:out, camunda:in:businessKey, camunda:executionListener, camunda:field }
    at Validator._logError (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:239:13)
    at Validator._validateProperty (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:197:18)
    at Array.filter (<anonymous>)
    at Validator._validateProperties (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:161:38)
    at Validator._validateTemplate (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:114:17)
    at Validator.add (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:54:20)
    at Array.forEach (<anonymous>)
    at Validator.addAll (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:39:17)
    at ElementTemplatesLoader.setTemplates (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/ElementTemplatesLoader.js:59:35)
    at BpmnEditor.loadTemplates (webpack-internal:///./src/app/tabs/bpmn/BpmnEditor.js:417:21) [ error ]
This error may be the result of a plug-in compatibility issue. [ info ]
Disable plug-ins (restarts the app) [ info ]
[/Users/maciej/Desktop/diagram_1.bpmn] invalid property.binding type <undefined>; must be any of { property, camunda:property, camunda:inputParameter, camunda:outputParameter, camunda:in, camunda:out, camunda:in:businessKey, camunda:executionListener, camunda:field }
    at Validator._logError (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:239:13)
    at Validator._validateProperty (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:197:18)
    at Array.filter (<anonymous>)
    at Validator._validateProperties (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:161:38)
    at Validator._validateTemplate (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:114:17)
    at Validator.add (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:54:20)
    at Array.forEach (<anonymous>)
    at Validator.addAll (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/Validator.js:39:17)
    at ElementTemplatesLoader.setTemplates (webpack-internal:///../../bpmn-js-properties-panel/lib/provider/camunda/element-templates/ElementTemplatesLoader.js:59:35)
    at BpmnEditor.loadTemplates (webpack-internal:///./src/app/tabs/bpmn/BpmnEditor.js:417:21) [ error ]
This error may be the result of a plug-in compatibility issue. [ info ]
Disable plug-ins (restarts the app) [ info ]
```
</details>

__Steps to Reproduce__

1. Create a broken template
2. Open application
3. now this happens

__Expected Behavior__

The error should contain at least the id of the template. Path to the templates file would also be appreciated, if available.

__Environment__

- OS: MacOS 10.15
- Camunda Modeler Version: 4.4.0 and earlier

___

Related to https://github.com/bpmn-io/bpmn-js-properties-panel/issues/386

**Comment by MaxTru** [comment]
Should be resolved with https://github.com/bpmn-io/bpmn-js-properties-panel/issues/386

**Comment by pinussilvestrus** [comment]
Closed via 2e1ecc1bec2a78d9e910a018cfc146e74d400469

![image](https://user-images.githubusercontent.com/9433996/111440154-5cb74400-8706-11eb-8d42-60b1bb1c0824.png)
